.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
Look at stackoverflow's posting guide. Mainly the parts about minimal reproducible code snippet. We may be able to use the same kind of wording here.

Reply:
Fixed in bc300ed1c47c135ff2bc18ba31abdaf1a528ef46